openslides/users/serializers.py

```diff
@@ -79,7 +81,9 @@ def create(self, validated_data):
             validated_data['default_password'] = User.objects.generate_password()
         validated_data['password'] = make_password(validated_data['default_password'], '', 'md5')
         # Perform creation in the database and return new user.
-        return super().create(validated_data)
+        user = super().create(validated_data)
+        inform_changed_data(user)
```

Comment:
Please add Comment here that this autoupdate call is redundant. Please add issue that caching for new objects with many 2 many fields has to be refactored (or just fixed). Then you can add the new issue number here.